content/12.privacy.md

```diff
@@ -1 +1,18 @@
 ## Rule 10: Don't share models trained on sensitive data
+
+One of the greatest opportunities for deep learning in biology is the high dimensional nature of much of the biological data collected today. 
+
```

Comment:
Now I'm seeing an extra newline between lines of text.  Are you able to remove those?  Sorry if my initial request was unclear.